src/components/network-modification-tree-pane.js

```diff
@@ -152,9 +152,27 @@ export const NetworkModificationTreePane = ({
                         removeNotificationByNode(currentNodeRef.current?.id)
                     );
                 }
+                if (
+                    studyUpdatedForce.eventData.headers['nodes'].some(
+                        (nodeId) => nodeId === selectedNodeIdForCopy
+                    )
+                ) {
+                    setSelectedNodeIdForCopy(null);
+                    snackInfo(
+                        '',
```

Comment:
Add a message key not the message
Add the message content param not the header param

Reply:
See with @sBouzols 
Every where in existing code the first param of snackInfo, snackError, snackWarning is a not translated error message which mainly come from back services.
Only the header message could use a translation key (headerMEssageId) to compute translation.
Maybe we could create another task to make this evolution to add a translated text in the content body of the snackbar.